### DISCUSSION TITLE
Incident subscription method

### DISCUSSION THREAD

**Discussion by seahindeniz** [community/community#128402] [npm]
Hello, could you please add Slack to the subscription methods on your status page, https://status.npmjs.org/ ? As an example, GitHub's status page has the same Atlassian integration, but it provides the ability to subscribe to a Slack channel where we keep up with our 3rd-party services and NPM is one of them. I hope you can enable it if that's not much trouble for the team. Thank you in advance.

**Comment by AKarbas** [comment]
Still undone but would be great if you could add it: https://support.atlassian.com/statuspage/docs/enable-slack-subscriptions/
